instructors.md

```diff
@@ -74,13 +74,20 @@ as long as learners using Windows do not run into roadblocks such as:
 
 *   Time estimates:
     *   @gvwilson: 3 hours
-
+	
+*	How to use the materials in the shell-novice repository (or, lesson planning)
+	* Use the `filesystem` directory for in-workshop exercises and live coding examples.  
+	The easiest way to get can get these files is by cloning the entire shell-novice directory.  Alternatives include creating a zipped copy of the `filesystem` directory and making that available to learners.  
```

Comment:
No need to create anything, since GitHub provides that zip for you under the [Download ZIP][1] button.  Although I personally think it's worth creating a new repository for a workshop that collects all the data learners will need in one repository (see efran/2015-04-09-UW#16, although we didn't end up using that approach for the UW workshop).

[1]: https://github.com/swcarpentry/shell-novice/archive/gh-pages.zip